bbc_restrictions/views/pricelist_view.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+		<record model="ir.ui.view" id="bbc_restrictions_product_pricelist_version_form_view">
+			<field name="name">product.pricelist.version.form.restriction.inherit</field>
```

Comment:
Looks like you mix tabs and spaces! Can you set your editor to use 4 spaces (not a tab) and reformat?

Reply:
You are right! My latest commit fixed the issue.